arangod/RocksDBEngine/RocksDBRecoveryManager.cpp

```diff
@@ -97,9 +99,13 @@ void RocksDBRecoveryManager::start() {
 /// parse recent RocksDB WAL entries and notify the
 /// DatabaseFeature about the successful recovery
 void RocksDBRecoveryManager::runRecovery() {
-  bool success = parseRocksWAL();
-  if (!success) {
-  }  // TODO what do we do if not successful?
+  auto res = parseRocksWAL();
+  if (res.fail()) {
+    LOG_TOPIC(FATAL, Logger::ENGINES)
+      << "Failed during rocksdb WAL recovery - "
+      << res.errorNumber() << " - " << res.errorMessage();
+    FATAL_ERROR_EXIT_CODE(TRI_EXIT_FAILED); //TODO - add exit recovery
```

Comment:
new code ideally shouldn't have TODOs. Can't we go on with using `TRI_EXIT_FAILED` and remove the TODO?